apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -28,7 +28,8 @@ const EVENTS = {
   PROGRESS_VIEWED: 'Section Progress Viewed',
   PROGRESS_TOGGLE: 'Section Progress Toggled',
   PROGRESS_CHANGE_UNIT: 'Section Progress Unit Changed',
-  PROGRESS_JUMP_TO_LESSON: 'Section Progress Jump to Lesson'
+  PROGRESS_JUMP_TO_LESSON: 'Section Progress Jump to Lesson',
+  FEEDBACK_SUBMITTED: 'Level Feedback Submitted'
```

Comment:
nit: do we think this should be in its own "section"? like "levels" or something? I can go either way and that's definitely not blocking this PR

Reply:
good call- added!